trompace/queries/controlaction.py

```diff
@@ -36,3 +38,14 @@ def query_controlaction(identifier: str):
     """
     query_ca = QUERY_CONTROLACTION_ID.format(identifier=identifier)
     return query_ca
+
+def query_controlaction_filter(filter: str, return_items_list: list = ["identifier", "name"]):
```

Comment:
I improved the way that we we pass in filters and return items, please take a look at an existing example and replicate this: https://github.com/trompamusic/trompace-client/blob/c21e635a46d84895f53b26d91c75a20040cf4745/trompace/queries/application.py#L8

especially:
 - call filter argument `filter_` because 'filter' is a keyword
 - don't specify a list in the `return_items` argument, set it to None and set a default in the body
 - need to call `make_filter` on the filter and pass the data to format_query correctly